pkg/ingress/model_build_target_group.go

```diff
@@ -214,6 +214,9 @@ func (t *defaultModelBuildTask) buildTargetGroupTargetType(_ context.Context, sv
 	case string(elbv2model.TargetTypeInstance):
 		return elbv2model.TargetTypeInstance, nil
 	case string(elbv2model.TargetTypeIP):
+		if t.disableAWSVpcCNIFlag {
```

Comment:
Why the feature gate is only applicable for Ingresses? The service type can also have IP target type.

Reply:
This was a miss from my side, fixed it.